metrics-facade-base/src/main/java/com/ringcentral/platform/metrics/producers/SystemMetricsProducer.java

```diff
@@ -36,19 +43,24 @@ public SystemMetricsProducer(
             threadsMetricNamePrefix,
             bufferPoolsMetricNamePrefix,
             classesMetricNamePrefix,
-            null);
+            null,
+                isDimensional);
```

Comment:
I'd suggest using the same indentation for all the parameters including the last one.

Reply:
fixed, thank you.